shared_bills/settings.py

```diff
@@ -20,7 +20,7 @@
 # See https://docs.djangoproject.com/en/3.0/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = 'e45i2uj&!e(8#-!_#n6z^vmj+o^dze8g^!jb!n2c4zxif&0r&@'
+SECRET_KEY = "e45i2uj&!e(8#-!_#n6z^vmj+o^dze8g^!jb!n2c4zxif&0r&@"
```

Comment:
DO NOT COMMIT SECRET KEY TO REPO
It should be obtained from environment variable